pkg/frontend/validate.go

```diff
@@ -76,6 +76,19 @@ func (f *frontend) validateOpenShiftUniqueKey(ctx context.Context, doc *api.Open
 // prevent mischief
 var rxKubernetesString = regexp.MustCompile(`(?i)^[-a-z0-9.]{0,255}$`)
 
+func validateNamespace(method, namespace string) error {
+	if namespace == "" ||
+		!rxKubernetesString.MatchString(namespace) {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "", "The provided namespace '%s' is invalid.", namespace)
+	}
+
+	if !utilnamespace.IsOpenShift(namespace) {
```

Comment:
Shouldn't this be 
```
if utilnamespace.IsOpenShift(namespace) {
```
The `IsOpenShift` function says `// IsOpenShift returns true if ns is an openshift managed namespace.`. Otherwise I think this behavior only allows access to managed components.

Reply:
I believe this is what we want, at least for the time being - Support will only be able to get pod metric data for workloads in the OpenShift namespaces.

I do expect a cluster-wide top pods Geneva Action to be created in the future, but lockbox will be a prerequisite as that is customer data.